server/serverAPI/controller/ShoppingListController.ts

```diff
@@ -204,13 +205,13 @@ export default class ShoppingListController extends BaseUserController {
             }
 
             if (!listHasItem) {
-                return this.sendError(404, res, "Use Add endpoint to add ingredient to the shopping list.");
+                return this.send(ResponseCodes.NOT_FOUND, res, "Use Add endpoint to add ingredient to the shopping list.");
             }
 
             let updatedUser = await this.requestUpdate(req.serverUser.username, user, res);
-            return this.sendSuccess(200, res, updatedUser.shoppingList);
-        } catch (e) {
-            return e;
+            return this.send(ResponseCodes.OK, res, updatedUser.shoppingList);
+        } catch (response) {
+            return response;
```

Comment:
the problem is... that's it returns response :rofl: and response that it will throw depends on what it has thrown... It's the drawback of trying to make everything concise... I have no workaround right now, maybe I will come up with better solution later on